File: controlStatements/forloop/nestedforloop/program14.java
/* 1 2 3 4
 * 2 3 4 5
 * 3 4 5 6
 * 4 5 6 7
 */
class four{
	public static void main(String[]args){
	
		for(int i=1;i<=4;i++){
			int num = i;
			for(int j=1;j<=4;j++){
				
				System.out.print(num +" ");
				num++;
			}
			System.out.println();
		}
	}
}

File: controlStatements/forloop/program14.c
/* Write a program to check the given number is automorphic or not */
class Demo{

	public static void main(String[]args){
	
		int num = 25;
		int temp = num;
		int temp1 = num*num;
		int sqr = 1;
		int count = 0;
		while(temp!=0){
		
			count++;
			temp = temp/10;
		}
		while(count!=0){
		
			sqr = sqr*10;
			count--;
		}
			rem = temp2%sqr;
			if(num==rem){
			
				System.out.println("automorphic number");
			}else{
			
				System.out.println("not an automorphic number");
			}
	}
}
